spec/backend_client_spec.lua

```diff
@@ -90,6 +94,17 @@ describe('backend client', function()
 
       assert.equal(200, res.status)
     end)
+
+    it('reports the call with the status', function()
+      local service = configuration.parse_service({ id = '42' })
+      local status = 200
+      stub(test_backend, 'send', function() return { status = status } end)
```

Comment:
Why this instead of the `test_backend.expect` ?

Reply:
Because in this case, we don't really care about what the backend client receives, we just want it to return a 200 so we can report it.

Reply:
Would something like `test_backend.expect { }.respond_with{status = 200}` work?